src/game/self_play.rs

```diff
@@ -35,16 +34,17 @@ impl SerializerBase {
             probs_vec[m.to_nn_idx()] = prob;
         }
 
+        let size = planes.len() * 8 + probs_vec.len() * 4 + 1;

```

Comment:
can you use library constants here instead of magic numbers?

Reply:
Rust is a little bit weird, in u64,i8 there is BITS constant and no BYTES one, and f32 doesn't have anything...
Its like the primitive name is so obvious, you don't really need these constant